rpm_spec/qubes-video-companion-dom0.spec.in

```diff
@@ -19,6 +19,11 @@ BuildRequires:  python3-setuptools
 BuildRequires:  python3-devel
 
 Requires:       gstreamer1-plugins-good
+Requires:       v4l-utils
+## Fedora 37 doesn't have libayatana-appindicator-gtk3
+Requires:       libappindicator-gtk3
+Requires:       notification-daemon
```

Comment:
The virtual name on Fedora is `desktop-notification-daemon`, bare `notification-daemon` package on Fedora is the GNOME's one...

Reply:
Done